src/platform/imx8m/include/platform/drivers/interrupt.h

```diff
@@ -105,15 +105,15 @@ int irqstr_get_sof_int(int irqstr_int);
  */
 #define IRQSTR_CHANCTL			0x00
```

Comment:
@paulstelian97 can you explain a little bit in the commit message the formula for computing new values. At least why are we replacing n with 5 - n.

Reply:
even checkpatch.pl cries for this

```
Commit ec0909f72c44 ("drivers: imx: interrupt: Adjust IRQ_STEER registers for i.MX8")
-------------------------------------------------------------------------------------
WARNING: Missing commit description - Add an appropriate one
```

Reply:
I will need to formulate this one better, let me think about how I should comment this.

Reply:
Updated.